lektor_shortcodes/__init__.py

```diff
@@ -34,5 +78,9 @@ class ShortcodesPlugin(Plugin):
     name = u'lektor-shortcodes'
     description = u'Shortcodes for Lektor.'
 
+    def on_markdown_config(self, config):
+        lexer = ShortcodeLexer(self.get_config())
```

Comment:
Would it not be easier to just have a ```if "global" not in sections: return``` here? That way we'd just not add the lexer if the global section was missing.

Reply:
Indeed! Amended… I’m using `section_as_dict` for the containment check though, because `config.keys()` yields an iterator in Python 3.